platforms/ios/lib/WysiwygComposer/Sources/HTMLParser/Extensions/NSAttributedString+Range.swift

```diff
@@ -17,13 +17,13 @@
 import Foundation
 
 /// Describe an eror occuring during HTML to attributed ranges manipulation.
-enum AttributedRangeError: LocalizedError, Equatable {
+public enum AttributedRangeError: LocalizedError, Equatable {
```

Comment:
why public?

Reply:
We do use these for logging purposes on the lib which is now in a separated package target